packages/playwright-ct-angular/registerSource.mjs

```diff
@@ -34,6 +40,8 @@ const __pwLoaderRegistry = new Map();
 const __pwRegistry = new Map();
 /** @type {Map<string, import('@angular/core/testing').ComponentFixture>} */
 const __pwFixtureRegistry = new Map();
+/** @type {WeakMap<import('@angular/core/testing').ComponentFixture, Record<string, import('rxjs').Subscription>>} */
+const __pwOutputSubscriptionRegistry = new WeakMap();
```

Comment:
We're storing the subscriptions for each output and each fixture here.
The key is the fixture itself. We could have used the `rootElement.id` but it would have required more important refactoring.
Or maybe we could merge `_pwFixtureRegistry` and `__pwOutputSubscriptionRegistry` in a single registry that contains both. What do you think?

Reply:
I would leave it as is. I'll retest it later, and if needed, i could proceed with a follow-up PR